### PR TITLE
chore: upgrade to latest next.js version

### PR DESCRIPTION
**Pull request by G3root**

### CODE REVIEW

File: apps/web/next.config.js [hunk 1]
```diff
@@ -37,6 +37,10 @@ const config = {
       },
     ];
   },
+  webpack: (config) => {
+    config.resolve.alias.canvas = false;
```
Comment (G3root) [inline apps/web/next.config.js:41]: fixes `Module not found: Can’t resolve ‘../build/Release/canvas.node’`

**Comment by G3root** [comment]
ping @Mythie

File: apps/marketing/next.config.js [hunk 2]
```diff
@@ -22,6 +22,11 @@ const config = {
       transform: 'lucide-react/dist/esm/icons/{{ kebabCase member }}',
     },
   },
+  webpack: (config) => {
+    // fixes: Module not found: Can’t resolve ‘../build/Release/canvas.node’
+    config.resolve.alias.canvas = false;
```
[the remark this answers is not in the document]
Comment (G3root) [reply apps/marketing/next.config.js]: done 👍🏼

**Comment by Mythie** [comment]
Yikes that was a rough one

/tip 40

**Pull request merged**